jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -698,6 +716,10 @@ public Action process() throws Exception
                         BufferUtil.toSummaryString(_content),
                         _lastContent,
                         _generator.getState());
+                if (BufferUtil.hasContent(_header))
+                    HttpConnection.this.bytesOut.addAndGet(_header.remaining() );
+                if (BufferUtil.hasContent(_content))
+                    HttpConnection.this.bytesOut.addAndGet(_content.remaining() );
```

Comment:
Should be:
```java
HttpConnection.this.add(BufferUtil.length(_header)+BufferUtil.length(chunk)+BufferUtil.length(_content));
```
Hmmm actually no.... this is in a loop, so don't we have to add how much was taken from each.... thinking....

Reply:
my suggested line is correct, but it should not go here, it should be in the FLUSH case below